.travis.yml

```diff
@@ -0,0 +1,12 @@
+
+language: node_js
+node_js:
+  - '12.14.1'
+cache:
+  directories:
+    - 'node_modules'
+script:
+  - npm test
+  - npm run coverage
```

Comment:
this will run your tests twice since the second script is running the first one. you should remain with one script